packages/ckeditor5-engine/src/model/utils/deletecontent.js

```diff
@@ -148,7 +148,13 @@ function getLivePositionsForSelectedBlocks( range ) {
 			// This is how modifySelection works and here we are making use of it.
```

Comment:
This, earlier and later parts of code would greatly benefit from comments showing what is going on / what is described as model (wrote in HTML-fashion).

Reply:
For example, I thought that it will be like this:

Before modify selection:

```
<paragraph>A[</paragraph><image></image><paragraph>]B</paragraph>
```

After:

```
<paragraph>A[]</paragraph><image></image><paragraph>B</paragraph>
```